### DISCUSSION TITLE
v1 Client Libraries HMAC Authentication Vulnerability

### DISCUSSION THREAD

**Discussion by Daniel-CISA-CVD** [WalletConnect/walletconnect-monorepo#4150] [Developer Support] [answered]
Can someone reach out to me at cvd@cisa.dhs.gov to discuss a reported vulnerability? We would like to share more information and get this addressed. Please include our case number in the subject "VU#374440"

**Comment by glitch-txs** [comment, marked as answer]
please report to security@walletconnect.com

Thanks!
